Column.js

```diff
@@ -0,0 +1,57 @@
+function Column(id, name) {
+	var self = this;
+	
+	this.id = id;
+	this.name = name || 'No name given';
+	this.element = createColumn();
+
+	function createColumn() {
+		
+		var column = $('<div class="column"></div>');
+		var columnTitle = $('<h2 class="column-title">' + self.name + '</h2>');
+		var columnCardList = $('<ul class="card-list"></ul>');
+		var columnDelete = $('<button class="btn-delete">x</button>');
+		var columnAddCard = $('<button class="column-add-card">Add card</button>');
+		
+		columnDelete.click(function() {
```

Comment:
instead of `click` method, you can use method `on` with first argument as name of event